pkg/health/health.go

```diff
@@ -95,7 +96,7 @@ func ReadinessHandler(probes []Probes, urlProbes []HTTPProbe) http.Handler {
 			}
 		}
 
-		setProbeResponse(w, http.StatusOK, "Service is ready")
+		setProbeResponse(w, http.StatusOK, constants.HTTPServerHealthReadinessPath)
```

Comment:
The 3rd argument is a message, which seems to be shown to the client fetching this endponit.
It seems to me that we are switching the message of the response here from `Service is ready` to `/health/ready`.
It does not seem like a big deal to make this change. But I'd be surprised to see a path in the body of the response. I think the body of the response should be what it currently is, or if you insist on changing this - it could be `ok` or something simple like that, indicating success.

Reply:
+1, please keep this as before.

Reply:
Oops thanks for catching this - this was an unintended change @draychev.